src/app/clients/add/formSections/AddressCard.tsx

```diff
@@ -25,18 +26,22 @@ const AddressCard: React.FC<CardProps> = ({ formErrors, errorSetter, fieldSetter
                     helperText={errorText(formErrors.addressLine1)}
                     onChange={onChangeText(fieldSetter, errorSetter, "addressLine1", true)}
                 />
+                <FormGap />
```

Comment:
Why not use a `flex` container with a gap instead of having a bunch of `<FormGap />`